docs/Local/installMPS.md

```diff
@@ -1,137 +1,127 @@
-The [Management Presence Server (MPS)](../Glossary.md#m) is a cloud-agnostic microservice that enables platforms featuring Intel® AMT to be managed over the internet. This section details how to set up the MPS on a development system.
 
-Figure 1 illustrates where MPS fits into the overall [microservice architecture.](../Glossary.md#m)
+- [Management Presence Server (MPS)](../Glossary.md#m)
+- [Remote Provisioning Server (RPS)](../Glossary.md#r)
+- [UI Toolkit](../Glossary.md#u)
+- [Remote Provisioning Client (RPC)](../Glossary.md#r)
 
-[![MPS](../assets/images/MPSDeployment.png)](../assets/images/MPSDeployment.png)
+Find details about architectural details, security issues, and more in [Microservices](../Microservices/overview.md).
 
-**Figure 1: Deploy Management Presence Server (MPS) on a development system.**
+## Network, Hardware, and Software Prerequisites
 
-## Clone the Repository
+**Before installing the toolkit, prepare the environment:**
 
-**To clone a repository:**
+1\. Configure a network that includes:
 
-1. Open a Command Prompt or Terminal and navigate to a directory of your choice for development. 
+-  A development system running Windows® 10 or Ubuntu* 18.04 or newer 
+-  One or more [Intel vPro®](https://www.intel.com/content/www/us/en/architecture-and-technology/vpro/what-is-vpro.html) device(s) to manage
 
-2. Clone the MPS repository and navigate to it with the following commands:
+!!! tip
+    A **flash drive** or equivalent means of transfer is necessary to copy the RPC to the managed device.
 
+!!! Info
+    Both development systems and managed devices must use a **wired (i.e., cable) connection** on the same network.
+
+2\. Install the prerequisite software on the development system:
+
+| Prerequisite Software | Purpose |
+| :----------- |  :--|
+| [Chrome* Browser](https://www.google.com/chrome) | Runs the WebUI console for configuring profiles and connecting devices | 
+| [git](https://git-scm.com/downloads)| Downloads the OpenAMT Cloud Toolkit repository | 
+| [Node.js* LTS 12.x.x or newer](https://nodejs.org/) | Installs the software | 
+
+## Download and Configure Software
+
+**To download the Open AMT Cloud Toolkit repository on the development system:**
+
+1\. Open a Command Prompt or Terminal and navigate to a directory of your choice for development. 
+
+2\. Clone the repository.
 ``` bash
-git clone https://github.com/open-amt-cloud-toolkit/mps.git && cd mps
+git clone --recursive https://github.com/open-amt-cloud-toolkit/open-amt-cloud-toolkit
+```
```

Comment:
after this step it will be run `make build` for linux/mac  
and it will be `./build.ps1` for windows. Where they will be prompted to enter their IP Address and then it will run all the install commands for them. 

After that step is complete -- should pick up at : 

## Start the MPS and RPS

Reply:
@rsdmike OK. That sounds optimal. My changes were just intermediate "guesses" that functioned as suggestions. I fully expected it to go from short->shorter->shortest once developers started working on a build and config modification scenario. Trying to be flexible.